ios/FluentUI/Vnext/AvatarGroup/AvatarGroup.swift

```diff
@@ -30,15 +30,15 @@ import SwiftUI
     var overflowCount: Int { get set }
     var style: MSFAvatarGroupStyle { get set }
 
-    func createAvatar(style: MSFAvatarStyle, size: MSFAvatarSize) -> MSFAvatarState
+    func createAvatar() -> MSFAvatarState
```

Comment:
MSFAvatarState allows the client code to change the size and style of the avatar. We should prevent that.
Please refer to the design approach on the PersonaButtonCarousel.